processor-api/src/main/java/com/synopsys/integration/alert/processor/api/NotificationProcessor.java

```diff
@@ -55,19 +62,33 @@ public NotificationProcessor(
 
     public void processNotifications(List<AlertNotificationModel> notifications, List<FrequencyType> frequencies) {
         try {
+            if (logger.isDebugEnabled()) {
+                List<Long> notificationIds = notifications.stream()
+                                                 .map(AlertNotificationModel::getId)
+                                                 .collect(Collectors.toList());
+                String joinedIds = StringUtils.join(notificationIds, ", ");
+                notificationLogger.debug("Processing notifications: {}", joinedIds);
```

Comment:
We may want some similar easy to search for keywords in the log messages like Start processing notifications:
Then we can search in the logs for a simple pattern `Start processing  notifications:`
 and a corresponding `Finished processing notifications:`. Just might make it easier to debug if we have a simple pattern to the debug messages.
```suggestion
                notificationLogger.debug("Start processing notifications: {}", joinedIds);
```